packages/google_maps_flutter/example/ios/Runner/AppDelegate.m

```diff
@@ -4,10 +4,11 @@
 
 @implementation AppDelegate
 
-- (BOOL)application:(UIApplication *)application
-    didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+- (BOOL)application:(UIApplication*)application
+    didFinishLaunchingWithOptions:(NSDictionary*)launchOptions {
   // Provide the GoogleMaps API key.
-  [GMSServices provideAPIKey:@"YOUR KEY HERE"];
+  NSString* mapsApiKey = [[NSProcessInfo processInfo] environment][@"MAPS_API_KEY"];
```

Comment:
This might be confusing for users, what do you think about still using the key "YOUR KEY HERE" if the env variable is not set?

Reply:
I'm thinking of making it so that in the future for local runs we require the api key to be set via env var. That way we do not need to edit these files, and flutter run will automatically pick them up. But for short term, i'm cool with making this be `"YOUR KEY HERE"`.